### ISSUE TITLE
JS client-side invoke not being received on server hub

### ISSUE THREAD

**Issue by tvsmk** [Azure/azure-signalr#1032]
### Describe the bug
I am using the `Microsoft.Azure.SignalR.AspNet` NuGet Package in two .NET Framework 4.7.2 applications. One is an MVC website where the client side JS runs, and the other is a Web API which contains the C# code for the hub. The website is hosted on a different domain to the Web API. 

The issue I am having is that when a client attempts to invoke a server hub method, nothing is happening. The breakpoint on the method in the hub code is not hit.

Both the client and server initialise SignalR using `app.MapAzureSignalR("Testing");`, with my connection string stored in the web config under `Azure:SignalR:ConnectionString`.

In the JS code I am connecting to the hub using the following code:
```javascript
var connection = $.hubConnection();
connection.logging = true;
connection.url = 'https://localhost:44311/signalr'; // the API server's domain
connection.disconnected(function() {
    connection.start();
});

var hubProxy = connection.createHubProxy('myHub');

// Listener for when hub calls context.Clients.All.testBroadcast("Hello from the server!"); This works as expected
hubProxy.on('testBroadcast', function(message){
    console.log('Received message: ' + message);
});

connection.start().done(function() {
    console.log('Connected!'); // This appears in the console
    testInvoke(hubProxy);
});

function testInvoke(proxy) {
    proxy.invoke('testInvoke', 'Testing123').done(function() { 
        console.log('Test invoke successful!');  // This does not appear in the console. However, the connection logs that the invoke call is made
    });
}
```

This is the code I have for the Hub on the API:
```c#
public class MyHub : Hub
{
    private static IHubContext context;
    public MyHub(IHubContext context)
    {
        MyHub.context = context;
    }

    public void TestBroadcast()
    {
        context.Clients.All.testBroadcast("Hello from the server!"); // Works perfectly
    }

    public void TestInvoke(string message)
    {
        // This should be invoked but the breakpoint never gets hit
        context.Clients.All.testBroadcast($"Your message was {message}");
    }
}
```

The connection to the hub works perfectly and the client can receive messages from the server when setting up event listeners on the hub proxy.

I am using Google Chrome to test and I cannot see any activity in the network tab in Developer Tools when the invoke call is made. 

One thing I have noticed is that after `connection.start()` has finished, `hubProxy.connection.url` is set to the Azure SignalR service's url instead of `https://localhost:44311/signalr` which it is set to when the proxy is first created. However, I think this is what is expected as messages from the server can still be received by the client.

Unfortunately I am not in a position to be able to switch to using .NET Core as a large codebase would need to be converted.

Am I missing a trick here? Thanks in advance for any help.

### Further technical details
- I am using `Microsoft.Azure.SignalR.AspNet` version 1.5.1 on both client and server
- I am using .NET Framework 4.7.2
- Client SignalR jQuery script is `jquery.signalR-2.4.1.js`
- jQuery version being used is 3.3.1

**Comment by vicancy** [comment]
Sorry for the late response, looks like this issue is completed missed. Is this still an issue for you?

**Comment by tvsmk** [comment]
Hi @vicancy,

That's okay, this was a while ago now so I can't remember too much about the process. It seems that I stopped using a static hub context as that was preventing the clients from receiving broadcasted messages. In my now working code, it passes in the context to the method that calls `context.Clients.All.someMethod(...)`. 

I also no longer use JavaScript to invoke the `TestInvoke` method, we now call it directly from C# code.

I think we had some problems after that but it was resolved by talking to Microsoft directly and they investigated the network traffic for us. Unfortunately I can't give you the details of that as it wasn't myself who communicated with them.

This might still be a problem, but as described we now have a way around it. Thanks
